R/calc_plan_stats.R

```diff
@@ -0,0 +1,88 @@
+calc_plan_stats <- function(redist_plans, redist_map, ...) {
+
+    redist_plans <- redist_plans %>%
+        mutate(total_vap = tally_var(redist_map, vap),
```

Comment:
In R packages, external code needs a namespace prefix. 
This should be `dplyr::mutate(...`